Vagrantfile

```diff
@@ -27,33 +27,46 @@ Vagrant.configure("2") do |config|
     vbox.cpus = 2
   end
 
+  config.vm.provider :vsphere do |vsphere|
+    # The following section sets login credentials for the vagrant-vsphere
+    # plugin to allow use of this Vagrant script in vSphere.
+    # This is not recommended for demo purposes, only internal testing.
+    config.vm.box_url = 'file://dummy.box'
+    vsphere.host =                  ENV['VSPHERE_HOSTNAME']
+    vsphere.compute_resource_name = ENV['VSPHERE_COMPUTE_RESOURCE_NAME']
+    vsphere.template_name =         ENV['VSPHERE_TEMPLATE_NAME']
+    vsphere.user =                  ENV['VSPHERE_USER']
+    vsphere.password =              ENV['VSPHERE_PASSWORD']
+    vsphere.insecure=true
+    vsphere.customization_spec_name = 'vagrant-vsphere'
+  end
+
+  master_ip = "172.24.197.101"
+
   # Set up each box
   (1..num_instances).each do |i|
     vm_name = "%s-%02d" % [instance_name_prefix, i]
     config.vm.define vm_name do |host|
-      domain = "mesos.test"
-
       # Provision the FQDN
-      host.vm.hostname = "%s.%s" % [vm_name, domain]
+      host.vm.hostname = vm_name
 
       # Assign IP and prepend IP/hostname pair to /etc/hosts for correct FQDN IP resolution
-      ip = "172.18.8.#{i+100}"
+      ip = "172.24.197.#{i+100}"
```

Comment:
Seems like checking if this is a jenkins build vs. a demo build could allow us to use these addresses, but that is starting to lean towards having multiple vagrant files.